create_table.rb

```diff
@@ -5,7 +5,11 @@
 
```

Comment:
At the top of this file is a `require 'sinatra'` line. Please remove it so that running this migration won't run sinatra too